asteroid/losses/multi_scale_spectral.py

```diff
@@ -0,0 +1,96 @@
+import torch
+import torch.nn as nn
+from torch.nn.modules.loss import _Loss
+from asteroid.filterbanks import STFTFB, Encoder
+from asteroid.filterbanks.inputs_and_masks import take_mag
+
+EPS = 1e-8
+
+
+class MultiScaleSpectral(_Loss):
+    """ Measure multi-scale spectral loss as described in [1]
+
+    Args:
+        n_filters (list): list containing the number of filter desired for
+            each STFT
+        windows_size (list): list containing the size of the window desired for
+            each STFT
+        hops_size (list): list containing the size of the hop desired for
+            each STFT
+
+    Shape:
+        est_targets (:class:`torch.Tensor`): Expected shape
+            [batch, , time]. Batch of target estimates.
+        targets (:class:`torch.Tensor`): Expected shape
+            [batch, 1, time]. Batch of training targets.
+        alpha (:class'float')
+
+    Returns:
+        :class:`torch.Tensor`: with shape [batch]
+
+    Examples:
+
+        >>> import torch
+        >>> from asteroid.losses import PITLossWrapper
+        >>> targets = torch.randn(10,2,32000)
+        >>> est_targets = torch.randn(10,2,32000)
+        >>> loss_func=PITLossWrapper(MultiScaleSpectral(), mode='wo_src')
+        >>> loss = loss_func(est_targets, targets)
+
+        >>> import torch
+        >>> targets = torch.randn(10,1,32000)
+        >>> est_targets = torch.randn(10,1,32000)
```

Comment:
Shouldn't this be `torch.randn(10, 32000)` as this is `'wo_src` mode + the tensors are unsqueezed in the forward?
This will also work, but I think it's confusing.